magefile.go

```diff
@@ -1718,23 +1718,37 @@ func createTestRunner(matrix bool, singleTest string, goTestFlags string, batche
 		ServiceTokenPath: serviceTokenPath,
 		Datacenter:       datacenter,
 	}
+
 	ogcProvisioner, err := ogc.NewProvisioner(ogcCfg)
 	if err != nil {
 		return nil, err
 	}
+
 	email, err := ogcCfg.ClientEmail()
 	if err != nil {
 		return nil, err
 	}
-	essProvisioner, err := ess.NewProvisioner(ess.ProvisionerConfig{
+
+	provisionCfg := ess.ProvisionerConfig{
 		Identifier: fmt.Sprintf("at-%s", strings.Replace(strings.Split(email, "@")[0], ".", "-", -1)),
 		APIKey:     essToken,
 		Region:     essRegion,
-	})
+	}
+
+	essProvisioner, err := ess.NewProvisioner(provisionCfg)
 	if err != nil {
 		return nil, err
 	}
 
+	provisionerMode := os.Getenv("TEST_INTEG_SERVERLESS")
```

Comment:
What about using `STACK="ess"` and `STACK="serverless"` as the variable to adjust the stack provisioner to use? In the case that `STACK=` is not defined it would default to `ess` as the stack.

I am looking to do the same for the instance provisioners with `INSTANCES="ogc"` and `INSTANCES="lxd"`.

Reply:
Ahhh, good idea.